File: Uri Online Judge/C#/1006.cs
using System;
using System.Globalization;

class URI
{
    static void Main(string[] args)
    {
        double num1 = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        double num2 = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        double num3 = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        double media = (num1*2 + num2*3 + num3*5) / 10;
        Console.WriteLine("MEDIA = {0}", media.ToString("0.0", CultureInfo.InvariantCulture));
    }
}

File: Uri Online Judge/Rust/1006.rs
use std::io::{stdin};

fn main() {
    let mut num1 = String::new();
    let mut num2 = String::new();
    let mut num3 = String::new();

    stdin().read_line(&mut num1).expect("");
    stdin().read_line(&mut num2).expect("");
    stdin().read_line(&mut num3).expect("");

    let num1: f64 = num1.trim().parse().expect("Num1 is not number");
    let num2: f64 = num2.trim().parse().expect("Num2 is not number");
    let num3: f64 = num3.trim().parse().expect("Num3 is not number");

    let media = (num1*2.0 + num2*3.0 + num3*5.0) / 10.0;

    println!("MEDIA = {:.1}", media);
}
